src/components/commons/ReviewCard/index.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames/bind';
+
+import { getFormatDate } from '@/utils';
+
+import { Avatar } from '@/components/commons/Avatar';
+import StarRating from '@/components/commons/StarRating';
+
+import { Reviewer } from '@/types';
+
+import styles from './ReviewCard.module.scss';
+
+const cx = classNames.bind(styles);
+
+type ReviewCardProps = {
+  user: Reviewer;
+  rating: number;
+  date: Date;
+  content: string;
+};
+
+const ReviewCard = ({ user, rating, date, content }: ReviewCardProps) => {
+  return (
+    <article className={cx('review-card')}>
+      <div className={cx('review-card-profile')}>
+        <Avatar size='small' profileImageUrl={user.profileImageUrl} />
+        <div className={cx('review-card-info')}>
+          <h1 className={cx('review-card-info-reviewer')}>{user.nickname}</h1>
+          <div className={cx('review-card-info-rating')}>
+            <StarRating size='small' rating={rating} />
+            <span className={cx('review-card-info-date')}>{getFormatDate(date)}</span>
+          </div>
+        </div>
+      </div>
+      <p className={cx('review-card-comment')} dangerouslySetInnerHTML={{ __html: content }}></p>
```

Comment:
```suggestion
      <p className={cx('review-card-comment')}>{content}</p>
```
댓글의 내용을 보여주는 부분이니까 위와 같이 구현할 수 있을 거 같은데
`dangerouslySetInnerHTML={{ __html: content }}` 이걸 사용하신 이유가 궁금합니다

Reply:
teatarea에 줄바꿈을 사용할 경우 <br /> 태그가 함께 받아지는데요, 그걸 처리해서 보여주기 위해 dangerouslySetInnerHTML 속성을 사용했습니다.
`content: "직장인, 성인으로 이루어져 있어서 좋았습니다.<br /> 프로 칼퇴러가 됩니다ㅋㅋㅋㅋ 오프라인으로 정모도 있으니 참여 많이 해주세요!!"`

Reply:
```suggestion
      <div className={cx('review-card-comment')}>
        {content.split('\n').map((content, index) => (
        <p key={index}>{content}</p>
        ))}
      </div>
```
줄바꿈을 하면 아마 \n으로 받아지는데 만약 그게 맞다면
위와 같은 방법으로 수정하면 좋을 거 같습니다

Reply:
개행을 replace(/(?:\r\n|\r|\n)/g, '<br>') 변환한 뒤 HTML을 직접 삽입하는 방식을 사용했었습니다.
위와 같은 방법이 더 좋은 방법인가요? 제가 한 가지 방법밖에 몰라서요!

Reply:
`content : "안녕하세요\n반갑습니다"`
이런 식으로 리뷰 데이터가 오면 \n 부분을 기준으로 split해서 새로운 배열을 리턴 받은 후
배열의 요소들을 p 태그에 하나씩 담아서 보여주는 방법인데 저도 이거 밖에 몰라서 뭐가 더 좋은 방법인지는 모르겠네요

Reply:
dangerouslySetInnerHTML 사용 시 보안 이슈가 있네요! 
이번에 현기님 방식으로 적용해 보겠습니다!